This patch inserts a missing definition of INFINITY to Nelly Moser's encoder.

diff -Naur mplayer-trunk/libavcodec/nellymoserenc.c mplayer-trunk-new/libavcodec/nellymoserenc.c
--- mplayer-trunk/libavcodec/nellymoserenc.c	2009-01-04 01:14:25.000000000 +0100
+++ mplayer-trunk-new/libavcodec/nellymoserenc.c	2009-01-04 01:35:25.000000000 +0100
@@ -45,6 +45,10 @@
 #define POW_TABLE_SIZE (1<<11)
 #define POW_TABLE_OFFSET 3
 
+#if !defined(INFINITY) && defined(HUGE_VAL)
+#define INFINITY HUGE_VAL
+#endif
+
 typedef struct NellyMoserEncodeContext {
     AVCodecContext  *avctx;
     int             last_frame;
